standard/lexical-structure.md

```diff
@@ -1004,7 +1004,7 @@ right_shift_assignment
 
 ### 6.5.1 General
 
-The pre-processing directives provide the ability to skip conditionally sections of compilation units, to report error and warning conditions, and to delineate distinct regions of source code.
+Pre-processing directives provide the ability to skip conditional sections of compilation units, report error and warning conditions, and delineate distinct regions of source code.
```

Comment:
I suspect this is two words in the wrong order rather than a misspelling:

```suggestion
The pre-processing directives provide the ability to conditionally skip sections of compilation units, to report error and warning conditions, and to delineate distinct regions of source code.
```

which I think make more sense and is what the text used to be – I don’t know offhand when the two words were swapped around.

As to dropping the The/to/to I'll leave that to a grammarian, though the text has been that way for a long time.

Reply:
That was my initial reaction when looking at this on a mobile earlier. I personally prefer "to conditionally skip" over either of the other options, despite being a split infinitive.

I'm fine with dropping "The" at the start of the sentence though.

Reply:
FYI: Checked back and the two words were swapped around between v2 & v3.

Reply:
Now that I read it aloud, the suggestion made by sounds better I think